controllers/space/space_controller_test.go

```diff
@@ -88,8 +88,8 @@ func TestCreateSpace(t *testing.T) {
 				nsTmplSetAssertion := nstemplatetsettest.AssertThatNSTemplateSet(t, test.MemberOperatorNs, "oddity", member1.Client).
 					Exists().
 					HasTierName(base1nsTier.Name).
-					HasClusterResourcesTemplateRef("base1ns-clusterresources-123456new").
-					HasNamespaceTemplateRefs("base1ns-code-123456new", "base1ns-dev-123456new", "base1ns-stage-123456new")
+					HasClusterResourcesTemplateRef("base1ns-clusterresources-123456new-cr").
+					HasNamespaceTemplateRefs("base1ns-code-123456new-cr", "base1ns-dev-123456new-cr", "base1ns-stage-123456new-cr")
```

Comment:
just curious - why did you had to add all the `-cr` suffixes in the tests? Is it because the status revisions constants were created with this new name or some other reason I'm missing?

Reply:
our end goal would be that we would have cr names as value in status.revisions map. so i tried to have different values than the key (templaterefs) in the map 
see the discussion here 
https://github.com/codeready-toolchain/host-operator/pull/1135#discussion_r1948735245

Reply:
let's change the suffix to `-ttr` to make it more clear

Reply:
updated it now PTAL

Reply:
I see , so you wanted to have TTR cr names and not TierTemplate names. Got it, thanks !